src/helm/benchmark/scenarios/lmkt_eclektic_scenario.py

```diff
@@ -0,0 +1,75 @@
+"""A multilingual closed-book QA (CBQA) dataset that Evaluates
+Cross-Lingual Knowledge Transfer in a simple, black-box manner"""
+
+from typing import List
+from datasets import load_dataset
+
+from helm.benchmark.scenarios.scenario import (
+    Scenario,
+    Instance,
+    TEST_SPLIT,
+    Input,
+)
+
+SUPPORTED_LANGUAGES = {
+    "de": "German",
+    "en": "English",
+    "es": "Spanish",
+    "fr": "French",
+    "he": "Hebrew",
+    "hi": "Hindi",
+    "id": "Indonesian",
+    "it": "Italian",
+    "ja": "Japanese",
+    "ko": "Korean",
+    "pt": "Portuguese",
+    "zh": "Chinese",
+}
+
+
+class CulturalKnowledgeRememberingEclekticScenario(Scenario):
+    """Cultural values understanding evaluation based on Vietnam World Values Survey responses."""
+
+    name = "cultural_knowledge_remembering_eclektic"
+    description = "Evaluates the cross-lingual knowledge transfer ability of LLMs"
+    tags = ["lmkt", "cultural_knowledge_remembering"]
+
+    def get_instances(self, output_path: str) -> List[Instance]:
+
+        instances: List[Instance] = []
+
+        dataset = load_dataset(
```

Comment:
Optional: it is a good practice to set `cache_dir` to be within `output_path` so that the downloaded files are kept in the local directory and are easier to clean up ([example](https://github.com/stanford-crfm/helm/blob/75f7a4a8818f88fb7c1e67cf4b3a9c2f0641caf1/src/helm/benchmark/scenarios/gpqa_scenario.py#L47-L56))